labelbox/client.py

```diff
@@ -1211,3 +1212,27 @@ def _format_failed_rows(rows: List[str],
                     "Timed out waiting for get_data_rows_for_global_keys job to complete."
                 )
             time.sleep(sleep_time)
+
+    def get_slice(self, slice_id) -> Slice:
```

Comment:
When we introduce `CatalogSlice` and `ModelSlice`, and have functions in those respective children classes, it won't be possible to do this:
```
catalog_slice = client.get_slice("catalog-slice-id")
catalog_slice.some_function_specific_to_catalog
```